ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/FilterChips.css

```diff
@@ -12,6 +12,6 @@
     display: none;
 }
 
-#workload-cves-table-toolbar .cve-severity-select .pf-c-select__toggle-wrapper {
+#workload-cves-table-toolbar .pf-c-select__toggle-wrapper {
```

Comment:
won't this affect other selects in the toolbar as well? e.g. the resources select